flow/environments/umich.py

```diff
@@ -13,10 +13,10 @@ class GreatLakesEnvironment(DefaultSlurmEnvironment):
 
     hostname_pattern = r"gl(-login)?[0-9]+\.arc-ts\.umich\.edu"
     template = "umich-greatlakes.sh"
-    _cpus_per_node = {"default": 36, "gpu": 40}
-    _gpus_per_node = {"default": 2}
-    _shared_partitions = {"standard", "gpu"}
```

Comment:
What happened to the standard partition?

Reply:
The standard partition as well as the gpu partition don't need specifying since their behavior matches the default.